spyderlib/restart_app.py

```diff
@@ -63,6 +65,47 @@ def is_pid_running(pid):
         return _is_pid_running_on_unix(pid)
 
 
+def launch_error_message(type_, error=None):

```

Comment:
`type_` seems an ugly error variable name to me. Perhaps `error_type` would be better?